sdk/core/azure-core/inc/azure/core/nullable.hpp

```diff
@@ -261,10 +261,72 @@ namespace Azure { namespace Core {
       return std::move(m_value);
     }
 
+    // observers
+
     /**
      * @brief `operator bool` on the condition of #Azure::Core::Nullable::HasValue.
      */
-    explicit operator bool() const noexcept { return HasValue(); }
+    constexpr explicit operator bool() const noexcept { return HasValue(); }
+
+    /**
```

Comment:
What motivated the addition of these new operators on Nullable?

Reply:
Because without them the code is clunky to interact with the underlying type.